### ISSUE TITLE
Limit to 5000 students - problem with PR

### ISSUE THREAD

**Issue by All3yp** [education/GitHubGraduation-2021#5501]
I opened a pull request earlier but ended up having problems with the merge and it was not applied, I spent some time trying to understand what happened and discovered that it was because I put special characters in the paper. I fixed the problem but spent hours without any validation again (I was waiting for review). I tried to close it and opened a new pull request that worked, but its current number is #5312 the previous one was #3179. Can this cause me to not be able to receive the swag?

**Comment by GabrielaBezerra** [comment]
Same problem here. My first PR was #3180, and minutes later, after seeing that my PR had been skipped by the Bot, I created a new one #5469 that has been approved and tagged ready for merge. Will I be able to receive the swag?

**Comment by Devansh2005** [comment]
@All3yp, @GabrielaBezerra  nothing gonna end up without swags. The total number of merged PR will be counted i think, not the opened PR.

**Comment by FabioMen10** [comment]
Same with me. Isn't merging more. Stopped on PR #501

**Comment by FabioMen10** [comment]
Sorry for my misunderstanding @Devansh2005 . Who approves the PRs isn't a bot.

**Comment by elisemoe** [comment]
For swag, we are counting the number of **merged** pull requests. 

There's both an automated and manual review of each PR. The automated review will verify file paths, character counts, etc. of your pull request. Once this check is complete, a "ready to merge" label is applied to your PR. Once this label is present, our team does a manual check to verify that everything meets the event Code of Conduct. This will take us some time to get through everyone but we're working hard to get there! 😁
